tests/unit/gist/controller-test.js

```diff
@@ -0,0 +1,42 @@
+import Ember from "ember";
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('controller:gist', {
+  // Specify the other units that are required for this test.
+  // needs: ['controller:foo']
+  needs: ['service:ember-cli']
+});
+
+// Replace this with your real tests.
```

Comment:
This test doesn't add value now that you added a real one below, can you delete it?